test/NuGet.Core.FuncTests/NuGet.Packaging.CrossVerify.Generate.Test/GenerateFixture.cs

```diff
@@ -1,45 +1,45 @@
 using System.IO;
 using NuGet.Common;
 using NuGet.Test.Utility;
+using Test.Utility.Signing;
 
 namespace NuGet.Packaging.CrossVerify.Generate.Test
 {
     public class GenerateFixture
     {
-        private const string PreGeneratePackageFolderName = "PreGenPackages";
-
         public string _dir;
 
         public GenerateFixture()
         {
-            _dir = CreatePreGenPackageForEachPlatform();
+            _dir = GeneratePackagesForEachPlatform();
         }
-        private static string CreatePreGenPackageForEachPlatform()
+
+        private static string GeneratePackagesForEachPlatform()
         {
             var root = TestFileSystemUtility.NuGetTestFolder;
-            var path = Path.Combine(root, PreGeneratePackageFolderName);
+            var path = Path.Combine(root, TestFolderNames.PreGenPackagesFolder);
             if (!Directory.Exists(path))
             {
                 Directory.CreateDirectory(path);
             }
 
-            //Create a folder for a each platform, under PreGenPackages folder.
+            //Create a folder for a each platform, under GeneratedPackages folder.
             //For functional test on windows, 2 folders will be created.
```

Comment:
is this comment about "2 folders" will be created still true. it isn't clear to my why that is true. should the comment be enhanced?